app/assets/javascripts/react_on_rails.js

```diff
@@ -127,14 +121,14 @@
     }
     console.error('message: ' + e.message);
     console.error('stack: ' + e.stack);
-    msg += 'Exception in rendering!\n' +
-      (e.fileName ? '\nlocation: ' + e.fileName + ':' + e.lineNumber : '') +
-      '\nMessage: ' + e.message + '\n\n' + e.stack;
-
-    var reactElement = React.createElement('pre', null, msg);
-
-    return properReact.renderToString(reactElement);
-  }
+    if (serverSide) {
```

Comment:
Using a boolean for serverSide as we don't need the rendering part unless we're on the server.